classifiers/dns.go

```diff
@@ -0,0 +1,30 @@
+package classifiers
+
+import (
+	"github.com/google/gopacket/layers"
+	"github.com/mushorg/go-dpi"
+)
+
+type DnsClassifier struct{}
+
+func (_ DnsClassifier) HeuristicClassify(flow *godpi.Flow) bool {
+	if len(flow.Packets) == 0 {
+		return false
+	}
+	for _, packet := range flow.Packets {
+		if layer := (*packet).Layer(layers.LayerTypeUDP); layer != nil {
+			srcPort := layer.(*layers.UDP).SrcPort
+			dstPort := layer.(*layers.UDP).DstPort
+			if srcPort != 53 && dstPort != 53 {
```

Comment:
I was thinking that, if a flow ends up having multiple packets (it currently doesn't), some will be responses coming from the server, so they will come from port 53. So, each packet must either be to port 53, or from it.